Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs

```diff
@@ -157,6 +158,11 @@ private void InitActionsGrid()
 
         private void AddToPageList()
         {
+            if (App.PageList == null)
```

Comment:
but why now it comes as null compared to before?

Reply:
Because It was being intialized in MainWindow.xaml earlier where we were holding Other Pages LIke SolutionPage/RunSetPage in teh PageList